.github/workflows/publish_dev.yaml

```diff
@@ -0,0 +1,45 @@
+name: Publish Hub Dev
+
```

Comment:
same comments above.
It is not show stopper but it would be better to consolidate into a single workflow and make `dev/prod` as strategy

Reply:
Consolidation `dev/prod` is not desirable because, in the future, they may have different required steps before the actual publishing since `dev` has high frequency and high allowed threshold for bugs/mistakes than the `prod`.

Reply:
Done